src/index.js

```diff
@@ -105,6 +107,11 @@ const yargsv = require("yargs")(process.argv.slice(2))
                     default: DEFAULT_MFA_TOKEN_ARN,
                     type: "string"
                 })
+                .options('preserve-credentials-cache', {
+                    describe: "Remove the AWS credentials cache folder when command is complete",
```

Comment:
The option name and description are currently at odds

```suggestion
                .options('preserve-credentials-cache', {
                    describe: "Retain the AWS credentials cache folder when command is complete; otherwise remove it",
```